src/components/NewWorkspaceModal.js

```diff
@@ -69,6 +71,9 @@ const NewWorkspaceModal = Utils.withDisplayName('NewWorkspaceModal', ({
         attributes: { description },
         copyFilesWithPrefix: 'notebooks/'
       }
+      if (bucketLocation) {
+        body.bucketLocation = bucketLocation
+      }
```

Comment:
It might be easier to read if we inlined this, like so

```javascript
const body = {
        namespace,
        name,
        authorizationDomain: _.map(v => ({ membersGroupName: v }), [...getRequiredGroups(), ...groups]),
        attributes: { description },
        copyFilesWithPrefix: 'notebooks/',
        ...(bucketLocation && { bucketLocation })
      }
```

Reply:
Done.